d2game/d2player/help_overlay.go

```diff
@@ -1,11 +1,11 @@
-// Package help contains the in-game diablo2 help panel
-package help
+package d2player
 
```

Comment:
I think what @brendanporter did with isolating individual panels to another sub-package is a smart idea; it keeps `d2player/` from becoming monolithic. I'm not sure about moving the help overlay back into `d2player/`.

@Ganitzsh Was this just to keep everything consistent?

@essial How do you feel about this?